qemu/tests/nvdimm.py

```diff
@@ -32,7 +32,7 @@ def run_guest_cmd(self, cmd, check_status=True):
         """
         status, output = self.session.cmd_status_output(cmd)
         if check_status and status != 0:
-            raise error.TestFail("Execute command '%s' failed, output: %s")
+            test.fail("Execute command '%s' failed, output: %s" % (cmd, output))
```

Comment:
I think , you need to add "test" in the __init__, and use "self.test.fail"  here.